### PR TITLE
Computing for SS HW 1 

### PR DESCRIPTION
**Pull request by hujinhui96**
So far so good, I finally figured out how to edit the file on R studio and pushed it to GitHub

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* It is a great practice to commit regularly like you did! Will definitely benefit you when working on a bigger and longer projects.
* You only completed 3 of the 4 required elements (Headers, Lists, Links). But other than that, great job :)

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |           |       X       |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* It is a great practice to commit regularly like you did! Will definitely benefit you when working on a bigger and longer projects.
* You only completed 3 of the 4 required elements (Headers, Lists, Links). But other than that, great job :)
* Make sure you commit files through RStudio, not uploading through github. This will result in the reduction of point in the ease-of-access score in assignments.

**Review by hehely — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |     x      |              |            |

- Code is neat and easy to understand

- Only three of the five required elements (headers, lists, and links) were used in the code, while the homework asked us to use at least four elements. You could use the emphasis element to emphasis something you think is important in your biography, or use the images element to insert a photo of yourself

**Comment by wanitchayap** [comment]
Just another note, it seems like you upload the README.md through Rstudio itself, not through the github page. This will result in the deduction of your ease-to-access evaluation in next assignments.

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# Computing for SS HW1 Bio
+
+## Background information
+My name is Jinhui Hu, 1st year grad student with psych major under MAPSS.
+I considered myself as a cellular backgrounded person, and I am planning to work in Dr.Hansel's lab: http://www.hansellab-uchicago.com/
```
Comment (wanitchayap) [inline README.md:5]: This link should be embedded in the text itself:
[Dr.Hansel's lab](http://www.hansellab-uchicago.com/)
And this is how you do it:
`[Dr.Hansel's lab](http://www.hansellab-uchicago.com/)`

**Pull request closed without merging**